### PR TITLE
Preliminary arc support

### PR DESCRIPTION
**Pull request by raboof**
Only non-rotated arcs for now.

cx and cy seem accurate, but the arc and sweep appear to have a pretty
large (rounding?) errors - do you see where these might be coming from?

### CODE REVIEW

**Comment by frmdstryr** [comment]
It seems pretty close to me.

image

**Comment by raboof** [comment]
@frmdstryr in that picture you had an arc that started at angle 0 and spanned the full 360 degrees - that indeed worked, but my algorithm was incorrect for most other angles since I mixed up 'phi' and 'theta'. I think it should be fine now though!

**Comment by raboof** [comment]
(please squash when merging)

File: inkcut/core/svg.py [hunk 1]
```diff
@@ -300,7 +354,11 @@ def parse(self, e):
             elif cmd == 'Q':
                 self.quadTo(*params)
             elif cmd == 'A':
-                self.arcTo(*params)
+                x1 = self.currentPosition().x()
+                y1 = self.currentPosition().y()
+                (rx, ry, x_axis_rotation, large_arc_flag, sweep_flag, x2, y2) = params
+
+                self.arc(x1, y1, rx, ry, x_axis_rotation, large_arc_flag, sweep_flag, x2, y2)
```
Comment (frmdstryr) [inline inkcut/core/svg.py]: Looking at this again... does `x_axis_rotation` need to be converted to radians here?

https://www.w3.org/TR/SVG/paths.html#PathDataEllipticalArcCommands "and an x-axis-rotation, which indicates how the ellipse as a whole is rotated, in degrees, relative to the current coordinate system".  

All of python's trig functions take radians https://docs.python.org/2/library/math.html#trigonometric-functions.

**Pull request merged**